csharp/ThePointClass/ThePointClass.Tests/Point.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace ThePointClass.Tests
+{
+    [TestClass]
+    public class PointTests
+    {
+        [TestMethod]
```

Comment:
this UnitTest is testing 2 different scenarios.
you may want to split this unit test into 2 different tests or you can use the [DataRow] attribute (it's available only with MSTest V2, since you are using MS Test).
http://stackoverflow.com/questions/1010685/does-mstest-have-an-equivalent-to-nunits-testcase

MS Test are not very popular though, you may also consider to switch to XUnit or NUnit. In that case you just need an adapter (just a nuget package) that will allow the default VS test explorer view to discover your XUnit/NUnit tests (if you have resharper installed you don't need the adapter, and you can run those tests through the resharper test explorer view)